lib/gems/pending/appliance_console.rb

```diff
@@ -307,7 +307,7 @@ module ApplianceConsole
             system_hosts.hostname = new_host
             system_hosts.set_loopback_hostname(new_host)
             system_hosts.save
-            LinuxAdmin::Service.new("network").restart
+            LinuxAdmin::Service.new("NetworkManager").restart
```

Comment:
@ailisp We are already using `hostnamectl` in [`LinuxAdmin::Hosts#hostname=`](https://github.com/ManageIQ/linux_admin/blob/master/lib/linux_admin/hosts.rb#L39). That's why @bdunne suggested just getting rid of this line.